docs/test-fixtures.md

```diff
@@ -101,7 +107,11 @@ struct Nttp_Fixture{
     int value = V;
 };
 
-TEMPLATE_TEST_CASE_METHOD_SIG(Nttp_Fixture, "A TEMPLATE_TEST_CASE_METHOD_SIG based test run that succeeds", "[class][template][nttp]",((int V), V), 1, 3, 6) {
+TEMPLATE_TEST_CASE_METHOD_SIG(
+    Nttp_Fixture,
+    "A TEMPLATE_TEST_CASE_METHOD_SIG based test run that succeeds",
+    "[class][template][nttp]",((int V), V), 1, 3, 6)
+{
```

Comment:
```suggestion
TEMPLATE_TEST_CASE_METHOD_SIG(
    Nttp_Fixture,
    "A TEMPLATE_TEST_CASE_METHOD_SIG based test run that succeeds",
    "[class][template][nttp]",
    ((int V), V),
    1, 3, 6) {
```

Reply:
@horenmar 

Separating each parameter on its own line, I totally get. :+1:  And it's up to you, of course, with the opening braces, but the reason I moved them to a separate line is that... well, for instance, this example now looks like this, [in the docs](https://github.com/catchorg/Catch2/blob/devel/docs/test-fixtures.md#signature-based-parametrised-test-fixtures):
```c++
TEMPLATE_PRODUCT_TEST_CASE_METHOD_SIG(
    Template_Fixture_2, 
    "A TEMPLATE_PRODUCT_TEST_CASE_METHOD_SIG based test run that succeeds", 
    "[class][template][product][nttp]", 
    ((typename T, size_t S), T, S),
    (std::array, Template_Foo_2),
    ((int,2), (float,6))) {
    REQUIRE(Template_Fixture_2<TestType>{}.m_a.size() >= 2);
}
``` 
It's almost impossible to pick out the code from the arguments, IMHO. This feels much more readable, at least to me:
```c++
TEMPLATE_PRODUCT_TEST_CASE_METHOD_SIG(
    Template_Fixture_2, 
    "A TEMPLATE_PRODUCT_TEST_CASE_METHOD_SIG based test run that succeeds", 
    "[class][template][product][nttp]", 
    ((typename T, size_t S), T, S),
    (std::array, Template_Foo_2),
    ((int,2), (float,6)))
{
    REQUIRE(Template_Fixture_2<TestType>{}.m_a.size() >= 2);
}
```

Reply:
I somewhat agree (usually I just leave a blank line at the start of the block in these cases), but this is what you should get if you run the code through clang-format configured with Catch2's settings.

Reply:
Hm. ...New settings? :grin:  (No, but seriously — works for me.)